components/dateinput/overview.md

```diff
@@ -10,119 +10,144 @@ position: 0
-| `Placeholder` | `string` | The `placeholder` attribute of the `input` element. The placeholder will appear if the component is bound to nullable DateTime object - `DateTime?`, but will not be rendered if the component is bound to the default value of a non-nullable DateTime object. The Placeholder value will be displayed when the input is not focused. Once the user focuses it to start typing, the Format Placeholder (default or [customized one](#format-placeholder)) will override the Placeholder to indicate the format the date should be entered in. |
-| `TabIndex` | `int` | The `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key. |
+| `Id` | `string` | The `id` attribute of the `input`. |
+| `Placeholder` | `string` | The [`placeholder` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder) of the `input`. The placeholder will appear only if the component is bound to nullable `DateTime?` object, the `Value` is `null` and the component is not focused. Once the user focuses it to start typing, the `FormatPlaceholder` (default or [custom one](#format-placeholder)) will override the `Placeholder` to indicate the expected date format. |
+| `TabIndex` | `int` | The `tabindex` attribute of the `input`. Use it to control the tabbing order of the inputs on the page. |
 | `ValidateOn` | `ValidationEvent` enum <br/> (`Input`) | The event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs) |
-| `Value` | `DateTime` or `DateTime?` | The value of the `DateInput`. Supports two-way binding. |
-
-### Styling and Appearance
+| `Value` | `DateTime` or `DateTime?` | The component value. Use with two-way binding or [`ValueChanged` event handler]({%slug components/dateinput/events%}#valuechanged). |
+| `Width` | `string` | The width of the Date Input. |
 
-The following parameters enable you to customize the appearance of the Blazor DateInput:
+### Typing User Experience
 
-| Attribute | Type and Default Value | Description |
-|----------|----------|----------|
-|`Class`| `string` |The CSS class that will be rendered on the `input` element|
-|`Width`|`string`|The width of the `DateInput`|
+The component provides multiple parameters, which control the [caret placement, two-digit year values and the auto-correct behavior of the Date Input]({%slug dateinput-keyboard-typing%}).
 
-You can find more options for customizing the Date Input styling in the [Appearance article]({%slug dateinput-appearance%}).
-
-@[template](/_contentTemplates/date-inputs/format-placeholders.md#format-placeholder)
-
-## DateTime and Nullable DateTime
-
-The behavior of the component will depend on the type of field it is bound to, and this can result in different user experience and values that it will output:
+### Styling and Appearance
```

Comment:
I suggest moving the `Class` and `Width` parameters to this section similar to https://docs.telerik.com/blazor-ui/components/datepicker/overview#styling-and-appearance

Reply:
This makes sense, but I have a few concerns about this -
* The Parameters subsections will become too many
* Class and Width are just two parameters that do not justify their own section.
* There are other parameters related to appearance, which are not mentioned in the Overview, so everything becomes a bit messy to me.